Makefile

```diff
@@ -18,14 +19,21 @@ all: ${BUILD_DIR}
 ${BUILD_DIR}: CMakeLists.txt test/CMakeLists.txt
 	cmake -B${BUILD_DIR} .
 
+vcpkg:
+	git submodule update --init --recursive
+
 dev:
 	# Only enable testing, not clang-tidy/sanitizers; the latter make the build
 	# too slow, and we can run them in CI
 	cmake -B${BUILD_DIR} -DTESTING=ON -DCMAKE_BUILD_TYPE=Debug .
 
-dev3:
+vcpkg-dev: vcpkg
```

Comment:
I would just update the `dev` target to use vcpkg as well. Might as well be consistent.

Reply:
I don't have a dog in this hunt.  I would consider not requiring the use vcpkg by default.  I have done what you requested.